articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -76,10 +80,44 @@ az ml data create -f [PATH_TO_YML_FILE] --workspace-name [YOUR_AZURE_WORKSPACE]
 [!Notebook-python[] (~/azureml-examples-main/sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb?name=upload-data)]
 ---
 
+If you already have your data present in an existing datastore and want to create data asset out of it, you can do so by providing path to the data in datastore as shown below, instead of providing path on your local machine.
+
+# [Azure CLI](#tab/cli)
+[!INCLUDE [cli v2](../../includes/machine-learning-cli-v2.md)]
+
+Create a .yml file with the following configuration.
+
+```yml
+$schema: https://azuremlschemas.azureedge.net/latest/data.schema.json
+name: fridge-items-images-object-detection
+description: Fridge-items images Object detection
+path: azureml://subscriptions/<fmy-subscription-id>/resourcegroups/<my-resource-group>/workspaces/<my-workspace>/datastores/<my-datastore>/paths/<path_to_image_data_folder>
+type: uri_folder
+```
+
+# [Python SDK](#tab/python)
+
+ 
+```Python
+my_data = Data(
+    path="azureml://subscriptions/<my-subscription-id>/resourcegroups/<my-resource-group>/workspaces/<my-workspace>/datastores/<my-datastore>/paths/<path_to_image_data_folder>",
+    type=AssetTypes.URI_FOLDER,
+    description="Fridge-items images Object detection",
+    name="fridge-items-images-object-detection",
+)
+```
+---
+
 Next, you will need to get the label annotations in JSONL format. The schema of labeled data depends on the computer vision task at hand. Refer to [schemas for JSONL files for AutoML computer vision experiments](reference-automl-images-schema.md) to learn more about the required JSONL schema for each task type.
 
 If your training data is in a different format (like, pascal VOC or COCO), [helper scripts](https://github.com/Azure/azureml-examples/blob/main/python-sdk/tutorials/automl-with-azureml/image-object-detection/coco2jsonl.py) to convert the data to JSONL are available in [notebook examples](https://github.com/Azure/azureml-examples/blob/sdk-preview/sdk/jobs/automl-standalone-jobs).
 
+
+### Using pre-labeled training data from Azure Blob storage
+If you have your labelled training data present in a container in Azure Blob storage, then you can access it directly from there by [creating a datastore referring to that container](how-to-prepare-datasets-for-automl-images.md#create-an-azure-blob-datastore). Once you have created a datastore in AML workspace, linked to a existing container in blob, you'll have to update authentication details for that datastore. You'll have to select subscription id, resource group and provide either Account Key or SAS token.
```

Comment:
Why do we need to do this step here ? 
I would suggest to remove this comment on authentication details and the screenshot below for the following reasons:
- Ideally this information should be in the "registering a datastore" page. 
- If you create a datastore using "account key" or "sas", this step is not needed. 
- Authentication with data might be a more complicated topic and there is some documentation around it here: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-administrate-data-authentication We may not cover all the scenarios with the document we currently have.

Reply:
I agree that we should remove it from here. But I think it should be specified somewhere. There is no clear info/screentshots anywhere which shows steps to register blob as datastore.  Should I later create a separate PR with authentication info [here](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-datastore?tabs=cli-identity-based-access%2Ccli-adls-identity-based-access%2Ccli-azfiles-account-key%2Ccli-adlsgen1-identity-based-access)  ?

Reply:
I think this should be owned by data team. Could you please reach out to Qianqian Nie and suggest to improve this part of their documentation.